index.html

```diff
@@ -4,10 +4,24 @@
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <title>Document</title>
+   <title>Toshia_moi_mentor_dorogoi</title>
+   <link href="src/bootstrap/bootstrap.min.css" rel="stylesheet">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet"> 
+   <link rel="stylesheet" type="text/css" href="dist/main.css">
 </head>
 <body>
-   <h1> header 1</h1>
-   
+   <header class="header">
+      <div class="header-top">
+         <nav class="nav">
```

Comment:
you don't need `ul -> li` inside your `nav` tag, you can have only `nav` and `a` inside that as all your navigation items, so please remove unnecessary code and make a small refactoring.

Reply:
you will have `nav.nav -> a.nav-item` and that is all

Reply:
done

Reply:
how it can be done if it's not, you still have the same html structure, `nav -> ul -> li -> a`, and you should refactor it to the `nav -> a`